### ISSUE TITLE
Docker not being detected

### ISSUE THREAD

**Issue by glenndierckx** [prabirshrestha/synology-nomad#21]
Hi

First of all, thanks a lot for this package. I wanted to explore the world of nomad a bit more. The installation itself was a breeze.

However, I'm unable to get docker being detected, even tho I followed the steps.

On my system, the nomad user, seems to be correctly created, verifying with `sudo synouser --get nomad` resulted in:
```
User Name   : [nomad]
User Type   : [AUTH_LOCAL]
User uid    : [258047]
Primary gid : [258047]
Fullname    : []
User Dir    : [/var/packages/nomad/home]
User Shell  : [/sbin/nologin]
Expired     : [false]
User Mail   : []
Alloc Size  : [110]
Member Of   : [3]
(999) synopkgs
(258047) nomad
(65538) docker
```

As you can see, the user is part of the 'docker' group.

running `sudo ls -l /var/run/docker.sock` results in:
```
srw-rw---- 1 root docker 0 Feb 26 11:49 /var/run/docker.sock
```

So the docker group seems to be correctly in there.

But sadly, when opening up the clients window in the nomad ui, docker seems to be undetected. Even tho I restarted the system a few times.

![image](https://github.com/prabirshrestha/synology-nomad/assets/1397473/ca982b1e-8a87-431d-8477-eb0507dbfc02)

**Comment by prabirshrestha** [comment]
You probably want to check nomad logs and see why it isn't working. Also verify the config has docker enabled.

**Comment by SirEdvin** [comment]
So, docker would not work, because of [this issue](https://github.com/hashicorp/nomad/issues/13669).

**Comment by glenndierckx** [comment]
So I'll need to find a way to run those commands for the DSM 7 package as root then.

**Comment by prabirshrestha** [comment]
Following the instructions in README.md should had worked. At least that is what works for me.

https://github.com/prabirshrestha/synology-nomad?tab=readme-ov-file#docker-support-in-nomad-for-synology

**Comment by glenndierckx** [comment]
Sorry for the delay, picked it up again.

Even tho I did follow the steps, create the group, added nomad and myself to the newly created docker group. I can verify that for my own user I can execute docker commands (while before I had to run it as root). In the nomad web ui, it kept on showing as docker not detected. When running the agent command as root, it did pick up docker, so I just ended up running it as root.
